shared-workers.md

```diff
@@ -27,11 +27,16 @@ There has been increasing [developer](https://github.com/GoogleChromeLabs/privac
 
 ## [Maintaining a Session](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/issues/124)
 
-A developer embeds chat.com on two of their sites site-a.com and site-b.com. chat.com uses Shared Workers to maintain a user session.
+A website, chat.example, offers a way to maintain active chat session between any sites the embed it using SharedWorkers.
+Before storage partitioning this was possible, but after storage partitioning the when chat.example is embedded on different sites and instanciates a SharedWorker it would no longer be shared.
```

Comment:
```suggestion
Before storage partitioning this was possible, but after storage partitioning the when chat.example is embedded on different sites and instantiates a SharedWorker it would no longer be shared.
```